zerver/lib/onboarding.py

```diff
@@ -334,7 +333,7 @@ def send_initial_realm_messages(realm: Realm) -> None:
     welcome_messages: List[Dict[str, str]] = [
         {
             "stream": Realm.INITIAL_PRIVATE_STREAM_NAME,
-            "topic_name": "private streams",
+            "topic_name": "private channels",
```

Comment:
Not marked for translation (should these be?) but noted when manually testing.

Reply:
I think they may be marked for translation in the revised onboarding, but not worth working on for now, given we're replacing it wholesale.